src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/GenerateReleaseContentFunction.cs

```diff
@@ -58,7 +58,8 @@ public class GenerateReleaseContentFunction
                     executionContext.FunctionName);
                 logger.LogError("{0}", e.StackTrace);
                 await UpdateStage(message, Failed,
-                    new ReleasePublishingStatusLogMessage($"Exception in content stage: {e.Message}"));
+                    new ReleasePublishingStatusLogMessage("Exception in content stage: " +
+                                                          $"{e.Message}\n{e.StackTrace}"));
```

Comment:
This is already getting logged into appp insights by the line above, `logger.LogError("{0}", e.StackTrace);`